devicetypes/smartthings/zwave-multi-button.src/zwave-multi-button.groovy

```diff
@@ -74,7 +74,7 @@ def initialize() {
 	if(childDevices) {
 		def event
 		for(def endpoint : 1..prodNumberOfButtons[zwaveInfo.prod]) {
-			event = createEvent(name: "button", value: "pushed", isStateChange: true)
+			event = createEvent(name: "button", value: "pushed", isStateChange: true, displayed: false)
```

Comment:
I think we still want the event to be displayed in at least the child device's history.

Reply:
This event was also duplicated and this is the reason why I added "displayed: false". 
Should I remove this change?

Reply:
Ok, but as I understand, in this function an event is created and then sent to child. I don't see where I use this event as the parent's event. 
I think this is a different situation than in CentralSceneNotification handling, where the event is both sent to child and returned as the parent's event. 
'initialize' is also called in the 'updated' and I think it may be the cause of the duplication. 
After moving child devices setup to "configure" function, those events were visible and not duplicated. 
What do you think about this solution?